### PR TITLE
Fix deployment status

### PR DESCRIPTION
**Pull request by kushalmalani**
## Description

> Describe the purpose of this pull request.

Fix Deployment Status

## 🎟 Issue(s)

Related #1084 #1102 #1085 #1189

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

**Inspect**
Screenshot 2023-06-26 at 16 14 25

**Deployment Wait**
Screenshot 2023-06-26 at 16 40 11

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `make test` before taking out of draft
- [x] Ran `make lint` before taking out of draft
- [x] Added/updated applicable tests
- [x] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: cloud/deployment/workerqueue/workerqueue.go [hunk 1]
```diff
@@ -245,7 +245,7 @@ func IsHostedCeleryWorkerQueueInputValid(requestedWorkerQueue *astro.WorkerQueue
 		errorMessage = fmt.Sprintf("max worker count must be between %d and %d", defaultOptions.MaxWorkerCount.Floor, defaultOptions.MaxWorkerCount.Ceiling)
 		return fmt.Errorf("%w: %s", errInvalidWorkerQueueOption, errorMessage)
 	}
-	if !(requestedWorkerQueue.WorkerConcurrency >= machineOptions.ConcurrentTasks) ||
+	if !(requestedWorkerQueue.WorkerConcurrency >= 1) ||
```
Comment (neel-astro) [inline cloud/deployment/workerqueue/workerqueue.go:248]: I am guessing this is not intentional and was just done for testing

**Review by neel-astro — COMMENTED** [review]
Looks fine apart from a minor nit

File: cloud/deployment/workerqueue/workerqueue.go (hunk 1 above)
Comment (kushalmalani) [reply cloud/deployment/workerqueue/workerqueue.go:248]: no it was...actually a bug fix for https://github.com/astronomer/astro-cli/issues/1267#issuecomment-1607054459

Comment (neel-astro) [reply cloud/deployment/workerqueue/workerqueue.go:248]: should we make it a constant then or leave some explanation?

Comment (kushalmalani) [reply cloud/deployment/workerqueue/workerqueue.go:249]: Added a comment.

**Review by neel-astro — APPROVED** [review]

**Review by sunkickr — APPROVED** [review]

**Pull request merged**